.github/ISSUE_TEMPLATE/blank_issue.md

```diff
@@ -0,0 +1,4 @@
+---
+name: Blank Issue
+about: Create an issue with a blank template.
+---
```

Comment:
The goal is to make it clear to people that GitHub issues are for pokecrystal, not their own use of it, so:
```suggestion
---
name: Update to pokecrystal
about: Suggest a possible change to pokecrystal itself.
---
```